src/authentication.apib

```diff
@@ -105,7 +106,7 @@ The required GET parameters are:
  - `client_id` - issued when you create your integration
  - `response_type` - must be "token" (instead of "code")
  - `state` - unique string to be passed back upon completion (optional)
- - `redirect_uri` - URL to redirect back to after authorization
+ - `redirect_uri` - URL to redirect back to after authorization - it is required to provide the allowed redirect URI(s) in the build section of our Marketplace
```

Comment:
PP: I agree with @jenssegers here to keep the info the same as with the auth code grant.